### PR TITLE
Category crud

### PR DESCRIPTION
**Pull request by Bankacha**
pozdrav guru, moze lagani check-out i approval, poy :*

### CODE REVIEW

File: src/admin/pages/CategoryModal.js [hunk 1]
```diff
@@ -0,0 +1,70 @@
+import { useEffect } from "react";
```
Comment (stveljko012) [inline src/admin/pages/CategoryModal.js]: Move this file from `Pages` dir.

Comment (Bankacha) [reply src/admin/pages/CategoryModal.js]: okay, my bad.

File: src/App.js [hunk 2]
```diff
@@ -32,6 +33,9 @@ function App() {
                     <Route path='/admin/products/:id'>
                         <ProductPage type={'edit'}/>
                     </Route>
+                    <Route path='/admin/category'>
```
Comment (stveljko012) [inline src/App.js]: rename this to `categories`

File: src/admin/pages/CategoryPage.js [hunk 3]
```diff
@@ -0,0 +1,63 @@
+import { useState } from "react";
+import { Row, Col, Card, Button } from "react-bootstrap"
+import { useSelector } from "react-redux";
+import { getCategories } from "../../store/selectors";
+import { CategoryModal } from "./CategoryModal";
+
+export const CategoryPage = () => {
```
Comment (stveljko012) [inline src/admin/pages/CategoryPage.js]: rename this to categories

File: src/store/actions/types.js [hunk 4]
```diff
@@ -6,4 +6,6 @@ export const chargeBill = 'CHARGE_BILL';
 export const setActiveCategory = 'SET_ACTIVE_CATEGORY';
 export const addNewProduct = 'ADD_NEW_PRODUCT';
 export const deleteProduct = 'DELETE_PRODUCT';
-export const editProduct = 'EDIT_PRODUCT';
\ No newline at end of file
+export const editProduct = 'EDIT_PRODUCT';
+export const createCategory = 'CREATE_CATEGORY';
+export const editCategory = 'EDIT_CATEGORY';
```
Comment (stveljko012) [inline src/store/actions/types.js]: delete same properties @Bankacha

**Review by stveljko012 — COMMENTED** [review]
fix these before merge

File: src/App.js (hunk 2 above)
Comment (Bankacha) [reply src/App.js]: resolved

File: src/admin/pages/CategoryPage.js (hunk 3 above)
Comment (Bankacha) [reply src/admin/pages/CategoryPage.js]: done

File: src/store/actions/types.js (hunk 4 above)
Comment (Bankacha) [reply src/store/actions/types.js]: your bad

**Review by Bankacha — COMMENTED** [review]
can you approve it babe?

**Review by stveljko012 — COMMENTED** [review]
Implement category deletation

**Comment by Bankacha** [comment]
> Implement category deletation

implemented and resolved bro!

**Review by stveljko012 — APPROVED** [review]
Good job!

**Pull request merged**